lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -555,10 +555,16 @@ def get_address(options)
         options[:billing_address] || options[:address]
       end
 
+      def add_token_transaction_type(xml, options)
```

Comment:
🤔 opinion:
It could be good to be consistent using parameters like the other methods.

Reply:
Looks like `options` is not necessary for the method.

Reply:
the name is because we are setting the `TokenTxnType` value, and you are right the options is not neccesary, I will update it, thanks